src/Administration/Resources/app/administration/src/module/sw-settings-language/page/sw-settings-language-detail/sw-settings-language-detail.scss

```diff
@@ -13,4 +13,13 @@
             right: 18px;
         }
     }
+
+    .sw-settings-language-detail__snippet-link-wrapper {
+        display: flex;
+        justify-content: flex-end;
+        margin-bottom: 8px;
+        max-width: 60rem;
```

Comment:
I'd like to avoid `rem` if possible, for the sake of consistency. Maybe we have some variable there already?